clients/src/main/java/org/apache/kafka/common/protocol/types/Type.java

```diff
@@ -222,9 +266,15 @@ public Long validate(Object item) {
             else
                 throw new SchemaException(item + " is not a Long.");
         }
+
+        @Override
+        public String documentation() {
+            return "Represents an integer between 0 and 2<sup>32</sup>-1 inclusive. " +
```

Comment:
"Represents a signed integer"

Reply:
I believe you wanted to write unsigned. As this information is already there in the type name, I would not like to repeat it.

Reply:
What about:
`"Represents a signed integer between 0 and 2<sup>32</sup>-1 inclusive. "`

This is a nit though, ignore and discard as necessary.